### PR TITLE
Gh actions

### PR DESCRIPTION
**Pull request by jburel**
switch to GH actions

### CODE REVIEW

**Comment by joshmoore** [comment]
Interesting. https://github.com/jupyterhub/repo2docker-action can even pre-cache images.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Build
```
Comment (sbesson) [inline .github/workflows/main.yml]: Is the idea to turn this into a template? If so, should this become `repo2docker.yml` with `name: repo2docker` or similar?

**Comment by jburel** [comment]
Closing and using the actions from https://github.com/ome/.github

**Comment by jburel** [comment]
@sbesson @joshmoore 
Tested the badge integration see https://github.com/jburel/omero-analysis-desktop/pull/1
I will move it to ``.github`` repo

**Pull request closed without merging**